cli/tests/pcluster/configure/test_pcluster_configure/test_subnet_automation_no_awsbatch_no_errors_with_config_file/pcluster.config.ini

```diff
@@ -5,14 +5,15 @@ aws_region_name = eu-west-1
 key_name = key1
 vpc_settings = default
 # Implied value
-# scheduler = sge
+scheduler = sge
 base_os = centos6
 # Implied value
 # compute_instance_type = t2.micro
 master_instance_type = t2.nano
 max_queue_size = 14
 initial_queue_size = 13
 maintain_initial_size = true
+additional_iam_policies = arn:aws:iam::aws:policy/AWSBatchFullAccess
```

Comment:
Why this policy with sge scheduler?

Reply:
It's like the region change bug, but this time it's due to the scheduler in the original config being awsbatch. This still applies: https://github.com/aws/aws-parallelcluster/pull/1915#discussion_r463367524

Reply:
Thanks for the clarification. These tests need to be definitely improved in a separate PR.